spec/decorators/content_items_decorator_spec.rb

```diff
@@ -4,7 +4,7 @@
   include Draper::ViewHelpers
 
   let(:organisation) { create(:organisation, title: "Organisation title", slug: "the-slug") }
-  let(:taxonomy) { create(:taxonomy, title: "taxonomy a") }
+  let(:taxonomy) { create(:taxonomy, title: "taxonomy a", content_id: "content-id-for-taxon123") }
```

Comment:
These can be changed to `build()` now as we don't exercise the DB anymore - and we only need the `title` populated.